Tests/Controller/AzineEmailControllerTest.php

```diff
@@ -0,0 +1,177 @@
+<?php
+namespace Azine\EmailBundle\Tests\Controller;
+use Azine\EmailBundle\DependencyInjection\AzineEmailExtension;
+use Azine\EmailBundle\Services\AzineEmailTwigExtension;
+use Azine\EmailBundle\Services\Pagination;
+use Azine\EmailBundle\Tests\FindInFileUtil;
+use Azine\EmailBundle\Services\AzineTemplateProvider;
+use Azine\EmailBundle\Entity\SentEmail;
+use Azine\EmailBundle\Controller\AzineEmailTemplateController;
+use Azine\PlatformBundle\Services\EmailTemplateProvider;
+use Symfony\Component\DependencyInjection\ContainerInterface;
+use Symfony\Component\HttpFoundation\File\Exception\FileNotFoundException;
+use Symfony\Component\HttpFoundation\ParameterBag;
+use Symfony\Component\HttpFoundation\Request;
+use Symfony\Component\HttpFoundation\Response;
+use Symfony\Component\HttpFoundation\Session\Session;
+use Symfony\Component\HttpFoundation\Session\Storage\MockFileSessionStorage;
+use Symfony\Component\Routing\RequestContext;
+use Symfony\Component\Security\Core\Authentication\Token\UsernamePasswordToken;
+use Symfony\Bundle\FrameworkBundle\Test\WebTestCase;
+use Symfony\Component\DomCrawler\Crawler;
+use Symfony\Bundle\FrameworkBundle\Client;
+use Azine\EmailBundle\Tests\TestHelper;
+use Doctrine\ORM\EntityManager;
+
+class AzineEmailControllerTest extends WebTestCase
```

Comment:
pagination should be tested as well. e.g. check it the number of pages is correct for the number of `sent_emails` and that opening the second page shows the second batch of `sent_emails`.